### DISCUSSION TITLE
Visual mark for Custom Tags

### DISCUSSION THREAD

**Discussion by Rep1ie** [zanllp/infinite-image-browsing#287] [Ideas]
I have 2 custom tags. Some images are tagged "like" and some are tagged "uploaded".

![Screenshot_2](https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/137343727/1db4997a-4532-4e22-aa79-94a9a91ea171)
![Screenshot_1](https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/137343727/cda942cd-45c7-4054-bf17-d062422727ad)

It's very time consuming to open the context menu every time to check which tag an image has.
So it would be nice if the browser could display which tag the image has, or at least have the tagged images somehow marked in the browser.

**Comment by zanllp** [comment]
I will consider this feedback and explore ways to improve the user experience, such as displaying the tags directly in the browser or marking the tagged images in some way. Please stay tuned for updates.

**Comment by zanllp** [comment]
Already supported. see #131 for more detail.
![image](https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/25872019/b2425cc4-ac82-4509-be89-84f87b24fe3e)
